src/components/tryout/shorten-link.tsx

```diff
@@ -110,7 +112,24 @@ export function TryOutTab() {
             </div>
           </CardContent>
           <CardFooter>
-            <Button onClick={handleLinkShortenGeneration}>Shorten Link</Button>
+            <Button
+              onClick={handleLinkShortenGeneration}
+              className={`text-sm text-white bg-black hover:text-green-600 hover:bg-green-50 active:scale-95 active:ring-4 active:ring-green-300 transition-transform duration-300 ease-in-out ${
+                loading
+                  ? "bg-green-300 cursor-not-allowed hover:bg-green-300"
+                  : ""
+              }`}
```

Comment:
There is a `cn` function in `lib/utils.ts` that is used to compound classnames, use that instead of this way. It will make it much cleaner

Reply:
Also the background changing to pale green does not look too great to me, probably just a lighter shade of the black, just to see a slight difference.